tests/base/test_DataMisfit.py

```diff
@@ -41,6 +44,8 @@ def setUp(self):
         self.prob = prob
         self.dobs = dobs
         self.dmis = dmis
+        import SimPEG
```

Comment:
is this used in the test at all?

Reply:
ah good catch. I was experimenting something while writing it and forgot to remove some part of that experimentation....

Reply:
fixed